main.css

```diff
@@ -1,30 +1,48 @@
 .naugtur .bc-chat-container {
     position: fixed !important;
     z-index: 11511 !important;
-    top: 45px !important;
-    left: 25%;
+    top: 50px !important;
+    right: 200px;
     background: #fff;
-    width: 50% !important;
-    height: calc(96% - 120px) !important;
+    width: 30% !important;
+    height: calc(100% - 110px) !important;
 }
+@media (max-width: 1240px) {
+    .naugtur .bc-chat-container {
+        right: 40px;
+    }
+}
+
+.naugtur #theme-header {
+    display: none !important;
+}
+
+.naugtur #page-wrapper.theme {
+    background: url(http://www.zastavki.com/pictures/originals/2013/Movies___Roger_Pirate_042988_.jpg) top center no-repeat #ddd;
+}
+
 .naugtur #stage {
     z-index: auto !important;
 }
+
 .naugtur .bc-chat-messages-container {
     top: 0 !important;
 }
+
 .naugtur #bc-chat-input-container,
 .naugtur .bc-chat-input,
 .naugtur .chat-activity.chat-message .inner {
-    width: calc(96% - 80px) !important;
+    width: calc(100% - 80px) !important;
```

Comment:
I tend to go 9X not 100% to leave out minor cross-browser issues with borders and crap. You sure it's going to work for everyone? I didn't have time (ok, will :P) to test it much, so I cut corners here.